dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventSearchParams.java

```diff
@@ -319,7 +317,7 @@ public List<QueryItem> getDataElementsAndFilters()
         return items;
     }
 
-    public EventSearchParams addDataElements( List<QueryItem> des )
+    public EventSearchParams addDataElements( Set<QueryItem> des )
```

Comment:
Does order matter? If so we should not make this change.

Reply:
The data elements is now actually a `LinkedHashSet,` so it should maintain the order. In the last commit, I removed the setter to not override the value, so now using the method `addDataElements`, where we just add elements to the existing collection should be enough.